community/teams/marketing.tt

```diff
@@ -23,6 +23,7 @@
       <li><a href="https://discourse.nixos.org/u/ysndr">Yannik Sander</a> (ysndr)</li>
       <li><a href="https://discourse.nixos.org/u/bpiv400">Barry Plunkett</a> (bpiv400)</li>
       <li><a href="https://discourse.nixos.org/u/tomberek">Thomas Bereknyei</a> (tomberek)</li>
+      <li><a href="https://discourse.nixos.org/u/bjth">Bryan Honof</a> (bjth)</li>
```

Comment:
It would be great if we could use this occasion to link the handle to the user profile so one can contact each team member directly. (see `documenation.tt`)

Reply:
Could you check the new commit? Is that what you meant?